### PR TITLE
Remove dokka from project and turn rx library into a basic Kotlin module

### PR DESCRIPTION
**Pull request by rodrigohenriques**
In order to reduce the size from library I decided to remove android plugin.

### CODE REVIEW

File: rx-library/build.gradle [hunk 1]
```diff
@@ -1,51 +1,15 @@
...
-  implementation 'com.android.support:appcompat-v7:26.1.0'
-
   implementation "io.reactivex.rxjava2:rxjava:2.1.7"
-  implementation "io.reactivex.rxjava2:rxandroid:2.0.1"
 
   testImplementation 'junit:junit:4.12'
 
-  androidTestImplementation 'com.android.support.test:runner:1.0.1'
-  androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+  compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```
Comment (rsicarelli) [inline rx-library/build.gradle]: Please use `implementation` instead `compile` here

File: rx-library/build.gradle [hunk 2]
```diff
@@ -1,51 +1,15 @@
...
-}
-
-dokka {
-  outputFormat = 'html'
-  outputDirectory = "$buildDir/javadoc"
-}
+apply plugin: 'kotlin'
 
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (rsicarelli) [inline rx-library/build.gradle]: Do you need that line? I think that you can remove it

Comment (rodrigohenriques) [reply rx-library/build.gradle]: Done!

File: rx-library/build.gradle (hunk 1 above)
Comment (rodrigohenriques) [reply rx-library/build.gradle]: Done!

**Pull request merged**